Update - Added Web link to page (#3932)
MicrosoftDocs/architecture-center: docs/data-guide/disaster-recovery/dr-for-azure-data-platform-architecture-content.md

>[!NOTE]
>Many customers still retain a large IaaS footprint. To provide recovery capabilities across IaaS, the key component to be added is [Azure Site recovery](/azure/site-recovery/site-recovery-overview). [Site Recovery](/azure/site-recovery/site-recovery-faq) will orchestrate and automate the replication of Azure VMs between regions, on-premises virtual machines and physical servers to Azure, and on-premises machines to a secondary datacenter.

> Within this foundational structure, Contoso has implemented the following elements to support its enterprise business intelligence needs, inline with the Microsoft Reference Architecture for [Analytics end-to-end with Azure Synapse](/azure/architecture/example-scenario/dataplate2e/data-platform-end-to-end).

Reviewer: ```suggestion
Within this foundational structure, Contoso has implemented the following elements to support its enterprise business intelligence needs, aligned to the guidance in [Analytics end-to-end with Azure Synapse](/azure/architecture/example-scenario/dataplate2e/data-platform-end-to-end).
```

> Within this foundational structure, Contoso has implemented the following elements to support its enterprise business intelligence needs, inline with the Microsoft Reference Architecture for [Analytics end-to-end with Azure Synapse](/azure/architecture/example-scenario/dataplate2e/#data-platform-end-to-end).

Reviewer: This link doesn't seem to work.  Can you please check for an updated link, @batemansogq.
